operator/src/main/java/oracle/kubernetes/operator/DomainSourceType.java

```diff
@@ -4,7 +4,33 @@
 package oracle.kubernetes.operator;
 
 public enum DomainSourceType {
-  Image,
-  PersistentVolume,
-  FromModel
+  Image {
+    @Override
+    public String getDefaultDomainHome(String uid) {
+      return "/u01/oracle/user_projects/domains";
```

Comment:
not appending uid ?

Reply:
We have never have in the past for Domain-in-image. I presume that this is because the image can only have one domain, but it's a question for @rjeberhard